ports/xmlsec/CMakeLists.txt

```
@@ -58,20 +58,20 @@ if(MSVC)
   add_compile_options(/wd4130 /wd4127 /wd4152)
 endif()
 
-add_library(libxmlsec ${SOURCESXMLSEC})
-add_library(libxmlsec-openssl ${SOURCESXMLSECOPENSSL})
+add_library(libxmlsec1 ${SOURCESXMLSEC})
```

Based on https://github.com/lsh123/xmlsec/blob/master/configure.ac#L2330, this should be

```suggestion
add_library(xmlsec1 ${SOURCESXMLSEC})
```
and the CMAKE_SHARED_LIBRARY_PREFIX / CMAKE_STATIC_LIBRARY_PREFIX should be left unset so it will produce:

> libxmlsec1.dylib
> libxmlsec1.a
> libxmlsec1.so
> xmlsec1.lib
> xmlsec1.dll